src/components/Modal/ValidateProposalModal.tsx

```diff
@@ -43,17 +48,18 @@ function ValidateProposalModal({ proposal, account }: { proposal: IProposal; acc
   const totalAmount = jobRateAmount + originServiceFee + originValidatedProposalFee + protocolFee;
 
   const onSubmit = async () => {
-    if (!walletClient || !publicClient) {
+    if (!walletClient || !publicClient || !tlClient) {
       return;
     }
+
     await validateProposal(
+      tlClient,
       chainId,
       walletClient,
       publicClient,
       proposal.service.id,
-      proposal.seller.id,
```

Comment:
@pranav-singhal this will not work

Reply:
Actually, the function provided by the sdk expects the proposal ID. Internally, the sdk extracts the seller id from it. So to consumer, they only need to provide the actual proposal id